### DISCUSSION TITLE
Add option to use sound effect instead of voice announcements

### DISCUSSION THREAD

**Discussion by TobiPeterG** [osmandapp/OsmAnd#20921] [Ideas]
This would be useful for e.g. recalculations of the route, driving faster than the speed limit etc.

Okay a short sound effect, maybe different effects for different contexts?

**Comment by DmitryAlexei** [comment]
Please try beep sounds
For more information, see https://osmand.net/docs/user/navigation/guidance/voice-navigation#beep-modes
